Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -36,6 +36,7 @@ protected function configure()
             ->setName('doctrine:database:create')
             ->setDescription('Creates the configured databases')
             ->addOption('connection', null, InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
+            ->addOption('if-not-exists', null, InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database already exists')
```

Comment:
@KingCrunch it is already fixed by https://github.com/doctrine/DoctrineBundle/pull/363